run.py

```diff
@@ -1,34 +1,39 @@
 #!/usr/bin/env python3
 import json
-import os, os.path as op
-from zipfile import ZipFile
+import os
+import os.path as op
```

Comment:
Is there a reason to `import os` and also `import os.path as op`? Why not just `import os` an then `os.path` in the script just as you would with `os.mkdirs`? Could you explain this to me?